docs/pages/docs/guides/auth-and-access-control.mdx

```diff
@@ -0,0 +1,532 @@
+
+const Post = list({
+  access: {
+    operation: {
+      // see above
+    },
+    filter: {
+      query: filterPosts,
+    }
+  },
+  fields: {
+    // see above
+  },
+});
+```
+
+See that object we return when the session data doesn't say there's a user with `isAdmin`? It's the same format as `where` arguments you can provide to the `posts` query through the GraphQL API:
+
+```graphql
+query {
+  posts(where: { isPublished: { equals: true } }) {
+    title
+  }
+}
+```
+
+!> An easy way to set up Access Control filters is to write them as queries in the GraphQL Playground, then test them to make sure they return the results you expect. When you're happy, copy and paste the `where` argument into your access control function.
+
+You can also return a boolean value to switch on or off access to a list entirely:
+
+- `true` allows access to all items in the list
+- `false` prevents access to all items in the list
+
+In the example above, if the user has `isAdmin` we return `true` allowing them to access all posts.
+
+!> Access control filters are **combined** with any filters provided through the GraphQL API, and there is no way to circumvent them when making HTTP Requests.
+
+Access control is always applied when querying or mutating items in a list, regardless of how GraphQL queries are written. For example, if we also had a **Tags** list that relates to **Posts**:
+
+```ts
+const Tag = list({
+  fields: {
+    label: text(),
+    posts: relationship({ ref: 'Post', many: true }),
+  },
+});
+```
+
+You can query all the posts linked to each tag, but the filters we've defined above on `Post` will still be applied.
+
+```graphql
+query {
+  tags {
+    posts {
+      title
+    }
+  }
+}
+```
+
+Unauthenticated users won't be able to find unpublished posts by querying the relationship field on tags, and you don't have to do anything special to make this work.
+
+### Operations
+
+Each specific operation that can be performed through the GraphQL API has corresponding access control you can provide. The four operations are:
+
+* Query
+* Create
+* Update
+* Delete
+
+Access control is operation-specific, so if you want to prevent _any_ changes being made to items in a list, you would need to provide settings for all the mutation operations: `create`, `update` and `delete`.
+
+#### Why it's `query` and not `read`
+
+We designed the Access Control API for lists to clearly reflect the GraphQL operations it affects. Because of the way GraphQL works, you can retrieve values from an item through any operation, even deletes. For example:
+
+```graphql
+query {
+  user(where: { id: 1 }) {
+    name
+  }
+}
+mutation {
+  updateUser(where: { id: 1 }) {
+    name
+  }
+}
+```
+
+Both of these operations will "read" the user's name. Having access control named "read" on the list might be confusing as to whether it affects just the query, or also the mutations, so we called it "query" to make it clearer that it only affects queries and **not** mutations.
```

Comment:
nit: Moving the "Having access control named "read" on the list might be confusing as to whether it affects just the query, or also the mutations, so we called it "query" to make it clearer that it only affects queries and **not** mutations." to beginning of this section instead of the end makes reading easier. 

The rest seems like useful, but secondary information that I can choose to ignore once i grep the main point. 

That's the point you're making, the rest is nuance that ideally

Reply:
👍 rewritten